app.py

```diff
@@ -5,8 +5,12 @@
 from docker_fargate.docker_fargate_stack import DockerFargateStack
 
 app = cdk.App()
-vpc_stack = VpcStack(app)
-docker_fargate_stack = DockerFargateStack(app, vpc=vpc_stack.vpc)
+
+env = app.node.try_get_context('env')
```

Comment:
It might be a good idea to check that we get a string here, it looks like this would be `None` without the command-line context parameter set.

Reply:
done